integration-cli/docker_api_create_test.go

```diff
@@ -113,3 +115,134 @@ func (s *DockerSuite) TestAPICreateWithInvalidHealthcheckParams(c *check.C) {
 	expected = fmt.Sprintf("StartPeriod in Healthcheck cannot be less than %s", container.MinimumDuration)
 	c.Assert(getErrorMessage(c, buf), checker.Contains, expected)
 }
+
+func (s *DockerSuite) TestAPICreateWithCustomMaskedPaths(c *check.C) {
```

Comment:
Can you move these to integration/container?

Reply:
ah so I had them there and they weren't working but will try again

Reply:
ok updated!